cypress-tests/cypress/e2e/PaymentUtils/Commons.js

```diff
@@ -1197,16 +1197,15 @@ export const connectorDetails = {
       },
     },
     CaptureGreaterAmount: {
-      Request: {
         Request: {
```

Comment:
can the same be fixed in `Paybox.js` (`L383` and `L405`) and also in `cypress-tests-v2/cypress/e2e/configs/Payment/Commons.js` (`L1916` and `L1218`)